framework/src/parser/Parser.C

```diff
@@ -1255,7 +1255,7 @@ Parser::extractParams(const std::string & prefix, InputParameters & p)
     }
     else
     {
-      if (p.isPrivate(it.first))
+      if (p.isPrivate(it.first) and !in_global)
```

Comment:
&& ??

Reply:
and is a valid cpp keyword

Reply:
we dont use it though. Use &&, it matches the local style